src/main/java/org/apache/sysds/common/Builtins.java

```diff
@@ -182,6 +182,7 @@
 	NORMALIZE("normalize", true),
 	NROW("nrow", false),
 	NAIVEBAYES("naivebayes", true, false),
+	NAIVEBAYESPREDICT("naivebayesPredict", true, false),
```

Comment:
For the enum type, you might want to use underscores for making it more readable, but please do it consistently for NAIVEBAYES as well.